node-js-client/scripts/tap-coverage.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable no-console */
+if (!process.env.NO_COVERAGE) {
+  var ass = require('ass').enable( {
```

Comment:
+1 for coverage, and I realize this is mimicing style in customs-server, but the `ass` module is pretty deprecated at the moment. Maybe have a look at how some other fxa modules were ported over to use `nyc` (which is `istanbul` based and can plug into the `coveralls` CI service).